src/NuGet.Core/NuGet.Packaging/PackageExtraction/PackageType.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Packaging
+{
+    public enum PackageSignType
+    {
+        Unknown,
```

Comment:
we should refrain from the practice to add unknown then it becomes so hard to understand in telemetry data. What is the case when it will be unknown? I think better option might be unsigned, authorSigneed, reproSigned, signed.